src/sentry/backup/services/import_export/impl.py

```diff
@@ -208,7 +208,9 @@ def import_by_model(
                 min_inserted_pk: int | None = None
                 max_inserted_pk: int | None = None
                 last_seen_ordinal = min_ordinal - 1
-                for deserialized_object in deserialize("json", json_data, use_natural_keys=False):
+                for deserialized_object in deserialize(
+                    "json", json_data, use_natural_keys=False, ignorenonexistent=True
+                ):
```

Comment:
> `ignorenonexistent=True`

Does this imply that the user is potentially running some unsupported version of sentry on their self-hosted platform with outdated models? I guess I'm not sure when this would happen that we don't want to outright reject or encourage the user to upgrade their sentry instance.

Reply:
Yes, this happens when we get imports from 'old' sentry. The customer that sales is helping now is on a 3 month old sentry instance, so within our supported range but still failing.

Instead of encouraging folks to maintain the DELETED_FIELDS and DELETED_MODELS lists we can skip over fields in the import file that are no longer present in our schema instead of hard failing.